### PR TITLE
add build warning for bad report refs

### PR DESCRIPTION
**Pull request by dannyroberts**

### CODE REVIEW

**Comment by dannyroberts** [comment]
@snopoke cc: @biy

**Comment by dannyroberts** [comment]
@snopoke @biyeun

File: corehq/apps/app_manager/models.py [hunk 1]
```diff
@@ -3378,6 +3378,35 @@ def uses_media(self):
         # for now no media support for ReportModules
         return False
 
+    def check_report_validity(self):
+        """
+        returns is_valid, valid_report_configs
+
+        If any report doesn't exist, is_value is False, otherwise True
```
Comment (snopoke) [inline corehq/apps/app_manager/models.py]: spelling `is_value` -> `is_valid`

**Comment by snopoke** [comment]
:+1:

File: corehq/apps/app_manager/models.py (hunk 1 above)
Comment (dannyroberts) [inline corehq/apps/app_manager/models.py]: thanks https://github.com/dimagi/commcare-hq/pull/8919

**Pull request merged**